### PR TITLE
Lint the coffeescript

### PR DESCRIPTION
**Pull request by cvkef**

### CODE REVIEW

**Comment by fotos** [comment]
120 chars is OK for modern projects. This is the max line length we also use in Ruby projects.

File: spec/reporter_spec.coffee [hunk 1]
```diff
@@ -179,7 +184,8 @@ describe 'Reporter', ->
 
         @subject.sendBeacon(@url, @simple_beacon_data).then callback, callback
 
-      it 'waits for transport element to finish before fulfilling the promise returned', (done)->
+      it 'waits for transport element to finish before fulfilling the promise
+        returned', (done)->
```
Comment (fotos) [inline spec/reporter_spec.coffee]: Breaking line like this is illegible and not pretty.

File: src/biskoto.coffee [hunk 2]
```diff
@@ -18,7 +18,9 @@ define ->
       return null
 
     @set: (name, value, options = {}) ->
-      document.cookie = "#{name}=#{encode( JSON.stringify(value) )}#{@_cookieOptions(options)}"
+      cookie_data =
+      document.cookie = "#{name}=#{encode(JSON.stringify(value))}" +
```
Comment (fotos) [inline src/biskoto.coffee]: Avoid string concatenations please.

**Comment by Zorbash** [comment]
Closing this PR, as it's outdated without motivation about updating it. We're in favor of linting, yet CoffeeLint doesn't seem like the right tool for the job.

**Pull request closed without merging**